data/account.go

```diff
@@ -313,6 +314,11 @@ func (acc *Account) Validate() *util.ValidationError {
 	if acc.Login == "" {
 		valErr.FieldErrors["login"] = "Please add login"
 	}
+	re := regexp.MustCompile("^[a-zA-Z0-9-_]*$")
```

Comment:
Shouldn't the last `-` be escaped, since it's not the range character but a literal dash?
`^[a-zA-Z0-9\-_]*$`